### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mrEckendonk**
## Hi 👋 Code reviewer

- Create a collection that keeps a list of books.
✅ Create a function to add a new book to the collection, with title and author.
✅ Create a function to remove a book from the collection (hint: you can use the array filter() method).
✅ Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
✅ A new book is added to the collection.
✅ The new book is displayed in the page.
- Make sure that when a user clicks the "Remove" button:
✅ The correct book is removed from the collection.
✅ The correct book disappears from the page.
✅ Make sure that data is preserved in the browser's memory by using localStorage.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  bookArr.push(myBooks);
+}
+
+function storeData() {
+  localStorage.setItem('bookCollection', JSON.stringify(bookArr));
+}
+
+function getData() {
+  bookArr = JSON.parse(localStorage.getItem('bookCollection'));
+}
```
Comment (gemmen29) [inline script.js]: - [x] Kindly try to fix that error 
image
this problem happens if the key `bookCollection` doesn't exist in local storage,
you can fix it by changing this line to be like this
```
bookArr = JSON.parse(localStorage.getItem('bookCollection')) || [ ] ;
```

File: README.md [hunk 2]
```diff
@@ -1 +1,98 @@
...
+
+![Desktop_Layout](assets/img/desktop.png)
+
+- Mobile Layout
+
+![Mobile_Layout](assets/img/mobile.png)
+
+
+
+Additional description about the project and its features.
```
Comment (gemmen29) [inline README.md]: - [x] [OPTIONAL] Kindly put a screenshot of your project or remove this section as it now makes your README not professional.

File: README.md [hunk 3]
```diff
@@ -1 +1,98 @@
...
+
+## Authors
+
+👤 **Author 1 **
+
+- GitHub: [@PinkMoon25](https://github.com/PinkMoon25/)
+- Twitter: [@twitterhandle](https://twitter.com/---)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/---/)
+
+👤 **Author 2 **
```
Comment (gemmen29) [inline README.md]: - [ ] [OPTIONAL] Kindly change the name of the authors with your name instead of `Author 1` and `Author 2`

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (mrEckendonk) [reply script.js]: Okay, will change that. Thanks for the info. Learned again something. 😄

File: README.md (hunk 2 above)
Comment (pinkmoon25) [reply README.md]: Thank you @gemmen29 we removed the section from README.

File: README.md (hunk 3 above)
Comment (pinkmoon25) [reply README.md]: hi, @gemmen29 we changed the author's name to ours.

**Review by ShahierNashaat — APPROVED** [review]
Hi @mrEckendonk and @PinkMoon25 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**